### ISSUE TITLE
Lost ability to export to csv, text and excel

### ISSUE THREAD

**Issue by ummspmacdon** [microsoft/azuredatastudio#17142]
Issue Type: <b>Bug</b>

All of sudden, the icons on right hand side of the result window are gone.  I can't high-light cells (any or all of them) and export to excell, csv or txt files.  I can't even right-click in a single cell and copy it's contents to cut and paste.  Truly frustrating.

Azure Data Studio version: azuredatastudio 1.32.0 (4a45ba7cf20dd4129f1a08e5e776dfb33e3d1d1e, 2021-08-16T18:08:28.086Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8365U CPU @ 1.60GHz (8 x 1896)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.81GB (7.36GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>

**Comment by kburtram** [comment]
@ummspmacdon are you seeing any errors in the "Help->Developer Tools" Console?  

I saw a similar issue a few weeks ago and deleting the cached application state resolved that issue (would be in C:\Users\[USERNAME]\AppData\Roaming\azuredatastudio on windows).  You may try that, though it would reset the application state to 1st-launch, so you may want to backup/restore the User/settings.json values, such as all the connection metadata.

**Comment by ummspmacdon** [comment]
Thanks, KB...tried that, but it didn't seem to help.  Anyone with any other thoughts?

I recall something somewhere about seeing a rectangle around the results window meant a setting was off.  But i can't seem to find out where I heard it or where that setting is.  When you select a column heading, the entire result block highlights with a rectangle.  See below:

![image](https://user-images.githubusercontent.com/38335914/134675458-6fb90039-f6fb-4af6-9db5-460ff059180c.png)

**Comment by kburtram** [comment]
@alanrenmsft do you have any suggestions of what could be happening here?

@ummspmacdon BTW, when you deleted your application state it should have reset your settings.  Did you happen to try see if the query editor worked correctly before restoring the settings?  That would be helpful to narrow the cause down to a particular setting if that was the cause.  You may also verify if things are working correctly with extensions disabled (there's a Disable All Installed Extensions command in the Extension Manager).

**Comment by alanrenmsft** [comment]
I'll take a look.

**Comment by ummspmacdon** [comment]
I wound up saving my settings.json file off and uninstalling ads completely.  Clearing out both the local and roaming AppData folders as well.  I reinstalled a newer version (1.32.0 vs. 1.22.1) as well.  With the completely clean install and the brand-new settings.json, a;; functionality (excel, csv, json, copy/paste etc.) was restored.  I replaced the new settings.json file with my previously saved off version and the functionality vanished once again.   .
I do have the recommended Extension packages installed, including SQL Prompt by Redgate, so it’s not in the extensions.  It definitely had something to do with my settings.  I’ll go back through each change I had different from default and see which one caused the issue.
I appreciate the help and the guidance.  I never would have thought to save off the settings.json file.  Nor would I have known about it.

Paul
